modular_skyrat/modules/customization/modules/client/preferences.dm

```diff
@@ -2229,8 +2229,8 @@ GLOBAL_LIST_INIT(food, list(
 							sextoys_pref = "Yes"
 							// User set ERP pref to "Yes", make the ERP button of the inventory visible and interactive again
 							if(user.hud_used)
-								for(var/atom/movable/screen/human/ERP_toggle/E in user.hud_used.static_inventory)
-									if(istype(E, /atom/movable/screen/human/ERP_toggle))
+								for(var/atom/movable/screen/human/erp_toggle/E in user.hud_used.static_inventory)
```

Comment:
Updating commented-out code, you mad lad 😎

Reply:
I'm thorough